### PR TITLE
perf: align icons and text vertical

### PR DESCRIPTION
**Pull request by ricosmall**
**1. To make all icons seems consistent.**

- Change icon `★` to `⭐️`
- Change icon `ⓘ` to `ℹ️`

**2. To make text vertically align left.**

Change template of `createTextNode`:

- Before:

```html
<tspan x="25" dy="${lineHeight}" class="stat bold">
    <tspan data-testid="icon" class="icon ${classname}">${icon}</tspan> ${label}:</tspan>
    <tspan data-testid="${id}" x="155" dy="0" class="stat">${kValue}</tspan>
```

- After:

```html
<tspan x="25" dy="${lineHeight}" class="stat bold">
    <tspan data-testid="icon" class="icon">${icon}</tspan>
    <tspan x="45"> ${label}:</tspan>
</tspan>
<tspan data-testid="${id}" x="155" dy="0" class="stat">${kValue}</tspan>
```

Preview:

- Before:

image

- After:

image

### CODE REVIEW

**Comment by anuraghazra** [comment]
Thank you for the improvement @ricosmall sorry i could not reach you because github was down

**Comment by anuraghazra** [comment]
Hey @ricosmall it looks good but as you can see since you shifted all the text to make them consistently aligned, if i don't enable show_icons then it shifts too

Preview (with your changes)-


Current

**Comment by infinity-plus** [comment]
> Also as you can see for some reason in windows it looks really weird 
Windows Emojis are ugly :(

**Comment by ricosmall** [comment]
@anuraghazra Just change the code to make text vertically align left, please merge it if you think it's OK.

And for unifying the icons, I think we can use the svg icons of GitHub itself, like below:

image

What do you think?

**Comment by anuraghazra** [comment]
Yes @ricosmall I also used github's svgs in GitHub extra pins.

**Comment by anuraghazra** [comment]
> @anuraghazra Just change the code to make text vertically align left, please merge it if you think it's OK.
> 
> And for unifying the icons, I think we can use the svg icons of GitHub itself so that it will looks like the same on Windows, Linux and MacOS. Demo below:
> 
> image
> 
> What do you think?

@ricosmall any luck with this? this is exactly what i need, do let me know your progress :D

**Comment by ricosmall** [comment]
> > @anuraghazra Just change the code to make text vertically align left, please merge it if you think it's OK.
> > And for unifying the icons, I think we can use the svg icons of GitHub itself so that it will looks like the same on Windows, Linux and MacOS. Demo below:
> > image
> > What do you think?
> 
> @ricosmall any luck with this? this is exactly what i need, do let me know your progress :D

@anuraghazra Finished now, have a look.

File: src/renderStatsCard.js [hunk 1]
```diff
@@ -52,31 +56,31 @@ const renderStatsCard = (stats = {}, options = { hide: [] }) => {
 
   const STATS = {
     stars: {
-      icon: "★",
+      icon: `<path fill-rule="evenodd" d="M8 .25a.75.75 0 01.673.418l1.882 3.815 4.21.612a.75.75 0 01.416 1.279l-3.046 2.97.719 4.192a.75.75 0 01-1.088.791L8 12.347l-3.766 1.98a.75.75 0 01-1.088-.79l.72-4.194L.818 6.374a.75.75 0 01.416-1.28l4.21-.611L7.327.668A.75.75 0 018 .25zm0 2.445L6.615 5.5a.75.75 0 01-.564.41l-3.097.45 2.24 2.184a.75.75 0 01.216.664l-.528 3.084 2.769-1.456a.75.75 0 01.698 0l2.77 1.456-.53-3.084a.75.75 0 01.216-.664l2.24-2.183-3.096-.45a.75.75 0 01-.564-.41L8 2.694v.001z"/>`,
```
Comment (anuraghazra) [inline src/renderStatsCard.js:59]: Maybe we could put it elsewhere as a file? it looks kinda messy inlined in here, any ideas @ricosmall

**Comment by anuraghazra** [comment]
@ricosmall I think this looks awesome :D thanks for the PR

**Pull request merged**